magento/pwa-studio
# [docs] peregrine talons overview

##  Description

Creates an unpublished doc topic about Peregrine talon. It is unpublished because talons are not currently available in 4.0.0.

## Related Issue





Closes #1756 

## Acceptance 

@jimbo 
@sirugh 

### Verification Stakeholders

@jimbo 
@sirugh 

### Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
2. Run the linter on the new file: `npm run lint _drafts/talons/index.md`
3. Verify no linting errors.

This topic is not published so there is no need to do an HTML preview.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
